initfiles/sbin/hpcc_setenv.in

```diff
@@ -81,6 +81,7 @@ IFS="${OIFS}"
 PATH_PREFIX=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^path *= *" | sed -e 's/^path *= *//'`
 
 export PID=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^pid *= *" | sed -e 's/^pid *= *//'`
+export LD_PRELOAD=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^java *= *" | sed -e 's/^java *=*//'`:$LD_PRELOAD
```

Comment:
I actually think it should probably be like that for the pid as well.  Going to test it and make sure.